includes/definitions/discovery/eatonups.yaml

```diff
@@ -1,8 +1,9 @@
-mib: XUPS-MIB:EATON-OIDS:EATON-EMP-MIB
+mib: XUPS-MIB:EATON-OIDS:EATON-EMP-MIB:UPS-MIB
 modules:
     os:
         hardware: XUPS-MIB::xupsIdentModel.0
         version: XUPS-MIB::xupsIdentSoftwareVersion.0
+        serial: UPS-MIB::upsIdentName.0
```

Comment:
Maybe this to cut it a bit;
`serial_regex: '/ID: (?<serial>[^ ]+)/'`

Reply:
Works perfectly ! thx !! 
![image](https://user-images.githubusercontent.com/84004634/144124950-682fd3e1-5341-4f53-a939-6122978f91d9.png)